libs/openFrameworks/3d/ofEasyCam.h

```diff
@@ -163,52 +160,70 @@ class ofEasyCam : public ofCamera {
 	/// Returns the area bounds used for mouse control.
 	ofRectangle getControlArea() const;
 	
+	
+	enum easyCamTransformType{
+		EASYCAM_TRANSFORM_NONE,
+		EASYCAM_TRANSFORM_ROTATE,
+		EASYCAM_TRANSFORM_TRANSLATE_XY,
+		EASYCAM_TRANSFORM_TRANSLATE_Z,
+		EASYCAM_TRANSFORM_SCALE
+	};
+
+	void addInteraction(easyCamTransformType type, int mouseButton, int key = -1);
+	void removeInteraction(easyCamTransformType type, int mouseButton, int key = -1);
+	bool getHasInteraction(easyCamTransformType type, int mouseButton, int key = -1);
```

Comment:
`getHas*` reads strangely -- can we just say `hasInteractionType(...)` or something?